rpcs3/Emu/Cell/Modules/cellMic.cpp

```diff
@@ -258,6 +258,29 @@ inline void microphone_device::variable_byteswap(const void* src, void* dst)
 	}
 }
 
+inline u32 microphone_device::convert_16_bit_pcm_to_float(const std::vector<u8>& buffer, usz num_bytes)
+{
+	constexpr usz float_buf_size = 2 * inbuf_size;
+	static_assert((float_buf_size % sizeof(u32)) == 0);
```

Comment:
We can now use f32 here so people don't get confused.
```suggestion
	static_assert((float_buf_size % sizeof(f32)) == 0);
```